services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -729,14 +729,7 @@ private string FixBlacksNewsgroupXml(string filePath, string xml)
             var match = _storyBody.Match(xml);
             if (match.Success)
             {
-                try
-                {
-                    xml = xml.Replace(match.Groups["body"].Value, this.Options.FailedStoryBody);
-                } catch (Exception ex)
-                {
-                    this.Logger.LogError(ex, $"FixBlacksNewsgroupXml - process XML throws exceptions. The XML is: {xml}");
```

Comment:
We don't need to push this kind of data to the logs.  All we need is the path to the file, then we can review the XML ourselves.